content/blog/azure-v6-release/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Pulumi Azure Classic Provider Version 6.0.0"
+date: 2024-09-24T15:04:00
+draft: false
+meta_desc: "Release of the next version of the Azure Classic Pulumi Provider"
+meta_image: meta.png
+authors:
+    - daniel-bradley
+tags:
+    - azure
+    - release
+---
+
+The latest major release of the Azure Classic Pulumi Provider is available now!
+Our 6.0 release contains the [latest upstream changes](https://github.com/hashicorp/terraform-provider-azurerm/releases/tag/v4.0.1) to keep you up-to-date along with the latest features and improvements from Pulumi.
+
+The Azure Classic provider can be used to provision any of the Azure resources available in the upstream provider.
```

Comment:
Does everyone have the context to know what the "upstream provider" is?

Reply:
Not sure ... don't know what else we'd call it. We use this term in issues quite a bit. Open to suggestions.

Reply:
Done 👍